Dockerfile

```diff
@@ -1,14 +1,17 @@
 # Copyright 2019 Adevinta
 
-FROM golang:1.21-alpine3.18 as builder
+FROM --platform=$BUILDPLATFORM golang:1.21-alpine3.18 as builder
 
 WORKDIR /app
 
 COPY go.mod .
 COPY go.sum .
 RUN go mod download
 COPY . .
-RUN CGO_ENABLED=1 go install -a -tags netgo -ldflags '-w' ./...
+
+ARG TARGETOS TARGETARCH
+WORKDIR /app/cmd/vulcan-results
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -tags netgo -ldflags '-w' .
```

Comment:
Is `-a` needed?

Reply:
Apparently not, a left-over from the previous go install. I removed it.